cmd/user-manager/main.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"log"
+	"os"
+	"strings"
+
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/percona/percona-xtradb-cluster-operator/cmd/user-manager/manager"
+	"github.com/pkg/errors"
+)
+
+func main() {
+	rootPass := os.Getenv("PXC-ROOT-PASS")
+	hosts := GetHostsFromEnvVar("PXC_CONNS")
```

Comment:
please reuse var from PXC StatefulSets
```suggestion
	hosts := GetHostsFromEnvVar("PXC_SERVICE")
```